cmd/poller/schedule/schedule.go

```diff
@@ -104,14 +106,13 @@ func (s *Schedule) IsStandBy() bool {
 	return s.standByMode
 }
 
-// SetStandByMode intiailzes StandbyMode: Schedule will suspend all tasks until
+// SetStandByMode initializes StandbyMode: Schedule will suspend all tasks until
 // the critical task t has succeeded. The temporary interval i will be used for
 // the task until Schedule recovers to normal mode.
 func (s *Schedule) SetStandByMode(t *task, i time.Duration) {
 	for _, x := range s.tasks {
 		if x.Name == t.Name {
 			s.standByTask = t
-			s.cachedInterval = t.interval // remember normal interval of task
```

Comment:
don't we need to cache this previous interval?

Reply:
the cache is created when the task is added to the schedule and then the cached value can never change